test/testSMSData.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    ToCountry: "US",
+    ToState: "CA",
+    SmsMessageSid: "SMd72ee1dd559b20b05a9a005df946b32f",
```

Comment:
data goes in the test/data folder, might not have merged that yet, but feel free to create it on this branch